test/integration/master/synthetic_master_test.go

```diff
@@ -314,6 +314,7 @@ func TestObjectSizeResponses(t *testing.T) {
 	expectedMsgFor2MB := `rpc error: code = ResourceExhausted desc = trying to send message larger than max`
 	expectedMsgFor3MB := `Request entity too large: limit is 3145728`
 	expectedMsgForLargeAnnotation := `metadata.annotations: Too long: must have at most 262144 characters`
+	expectedMsgForLargeAnnotation := `metadata.annotations: Too long: must have at most 262144 bytes`
```

Comment:
this should probably be replacing the line above?

Reply:
yeah, rebase error :-/